Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM eclipse-temurin:21.0.2_13-jdk-jammy
+
+# Update the APT cache
+# Prepare for Java download
+RUN apt-get update \
+    && apt-get upgrade -y \
+    && apt-get install -y --no-install-recommends
+COPY topicgenerator/target/topicgenerator*[^s].jar /home/topic-generator.jar
```

Comment:
What's the `[^s]` do?

Reply:
After a build, there are 2 JARs in the target directory:

topicgenerator-1.16.0-SNAPSHOT-sources.jar
topicgenerator-1.16.0-SNAPSHOT.jar

The `[^s]` is what I came up with to skip the sources JAR.